### PR TITLE
Add May 2024 minutes

### PR DESCRIPTION
**Pull request by hythloda**
Adds May 2024 minutes

### CODE REVIEW

File: minutes/2024-05-28.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+   </td>
+   <td>Microsoft Corporation
+   </td>
+  </tr>
+  <tr>
+   <td>Julia Ferraioli
+   </td>
+   <td>X
+   </td>
+   <td>Director of Program Management
```
Comment (juliaferraioli) [inline minutes/2024-05-28.md:39]: This should be "Amazon Web Services"

File: minutes/2024-05-28.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  <tr>
+   <td>Leslie Lamport 
+   </td>
+   <td>X
+   </td>
+   <td>Microsoft Corporation
+   </td>
+  </tr>
+  <tr>
+   <td>Julia Ferraioli
```
Comment (juliaferraioli) [inline minutes/2024-05-28.md:35]: Please lowercase my name

File: minutes/2024-05-28.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>Chris Newcombe
+   </td>
+   <td>X
```
Comment (juliaferraioli) [inline minutes/2024-05-28.md:21]: What do the "X"s indicate? If they mean something, please add it in the header row.

File: minutes/2024-05-28.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+**Governing Board Members In Attendance**
+
+<table>
+  <tr>
+   <td><strong>TLA+ and Linux Foundation Staff</strong>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
```
Comment (juliaferraioli) [inline minutes/2024-05-28.md:16]: These should be `<th></th>` to indicate they are headers.

File: minutes/2024-05-28.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+</table>
+
+<table>
+  <tr>
+   <td><strong>TLA+ and Linux Foundation Staff</strong>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
```
Comment (juliaferraioli) [inline minutes/2024-05-28.md:51]: These should be `<th></th>` to indicate they are headers.

**Review by juliaferraioli — CHANGES_REQUESTED** [review]
A few format and content changes needed.

File: minutes/2024-05-28.md (hunk 3 above)
Comment (hythloda) [reply minutes/2024-05-28.md:21]: Added the Present title. Thanks!

**Review by juliaferraioli — APPROVED** [review]
LGTM

**Comment by xxyzzn** [comment]
I posted the document "The Future of TLA+" on the TLA+ website and put a pointer to it on the site's News page.  I don't remember if we decided to post a pointer to it on the Google group.  I don't know if it would be a good idea to stir up discussion about this when there don't seem to be any need to.

Leslie

**Comment by lemmy** [comment]
Recent blog posts by long-time TLA+ users [1,2,3] have raised the question of whether TLA+ needs a different surface syntax.  This suggests that the TLA+ Foundation and the Specification Language Committee should more actively communicate their stance on this issue to balance the public discussion.

[1] https://protocols-made-fun.com/consensus/matterlabs/quint/specification/modelchecking/2024/07/29/chonkybft.html
[2] https://buttondown.com/hillelwayne/archive/an-idea-for-teaching-formal-methods-better/
[3] https://jack-vanlightly.com/analyses/2024/7/3/understanding-apache-paimon-consistency-model-part-3

/cc-ing @muenchnerkindl @muratdem @cnewcom for visibility

**Pull request merged**